app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from __future__ import annotations
+from app.cafe import Cafe
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    try:
+        for friend in friends:
+            try:
+                cafe.visit_cafe(friend)
+            except NotWearingMaskError:
+                masks_to_buy += 1
+    except VaccineError:
+        return "All friends should be vaccinated"
+    if masks_to_buy:
```

Comment:
better to use something obvious

Reply:
@Vasyl-Smutok what do you mean? in condition?

Reply:
`if masks_to_buy != 0:` or `if masks_to_buy >= 1:`

Reply:
I followed the checklist for this task - and there is stated that it's better not to compare with 0.